setup.py

```diff
@@ -173,7 +173,11 @@ def do_setup():
               'theano.misc': ['*.sh'],
               'theano.d3viz' : ['html/*','css/*','js/*']
           },
-          scripts=['bin/theano-cache', 'bin/theano-nose', 'bin/theano-test'],
+          entry_points={
+              'console_scripts': ['theano-cache = bin.theano_cache:main',
+                                  'theano-nose = bin.theano_nose:main',
+                                  'theano-test = bin.theano_test:main']
```

Comment:
Does that actually work when the current directory is not `Theano`?
I thought that `bin` here was interpreted as a package name, and that we would have to move the files inside of `Theano/theano`, so we could use `theano.<...>:main`.